text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+
+- legal expertise;
+- community building;
+- smart contract development (see VDAO);
+- long-term thinking;
+
+Successful implementation will require clear signals from the CasperLabs management team being transmitted to the core set of validators assisting in the incubation of the network.  If stakeholders can relatively quickly arrive at consensus on both the broad and finer strokes of a proposal, then the association  may be established in a timely manner, i.e. before main-net.  It may even be possible to establish an embryonic VDAO before main-net but more likely this would be released at a later date. 
+ 
+## Drawbacks
+
+[drawbacks]: #drawbacks
+
+1. The proposals outlined in this CEP are potentially problematic from a political & jurisdictional perspective.  The crypto space is unique in that attracts a fair number of non-regulated entities, such entities may very likely be put off by the idea of mandatory association membership.
+
+2. The notion that validators may be classified in some respect, e.g. regulated | certified | other, requires some form of arbiter.  This is problematic if, for example, the arbiter is found to be byzantine.  
+
+## Rationale and alternatives
+
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+In respect of establishing a validator association this is considered to be a partial formalisation of the hitherto proposed 'social consensus'.  It is the assertion of the CEP author that not visibly addressing AML/CFT concerns will result in a reduction in both the amount of capital and prestige of entities committed to the network.  
+
+The impact of not forming an association is that political cliques will inevitably form.  Such cliques focus upon short-medium term tactical objectives to the detriment of coherent long-term strategic objectives.  Furthermore those isolated from cliques, possibly due to simple issues regarding language & culture, may feel somewhat alienated and hence less committed to participating in the network over the long term. 
+
+## Prior art
+
+[prior-art]: #prior-art
+
+- Due to it's potential reach, the Libra project has been forced to think deeply upon the regulatory setting within which the network operators conduct business.  It has formed [an association](https://libra.org/en-US/white-paper/#the-libra-association) in order to navigate a complex regulatory landscape.
+- The Tezos network has a strong validator community that has effectively self organised over time.  
+
+## Future possibilities
+
+[future-possibilities]: #future-possibilities
+
+1.  Freeze accounts **provably** tainted to be associated with criminal activity.  The network's core account model could be updated to include a 'reject tainted accounts' setting.  If set to true then the network would implicitly reject transactions associated with tainted accounts.
```

Comment:
See above about handing the keys to the city over before they even got started on the contravallation.